documentation/proc-pages/physics-models/plasma_geometry.md

```diff
@@ -1,105 +1,682 @@
+  $$
+  $$
+   \delta_{95} = \delta / 1.5
+  $$
 
-- `ishape = 3` -- the Zohm ITER scaling is used to calculate the elongation (as for `ishape = 2` 
+---------------------------------------------------------------------
+- `ishape = 3` -- The Zohm ITER scaling[^3] is used to calculate the elongation (as for `ishape = 2` 
   above), but the triangularity at the 95% flux surface is input via variable `triang95`, and the 
   LCFS triangularity `triang` is calculated from it, rather than the other way round.
-  
-- `ishape = 4` -- the 95% flux surface values `kappa95` and `triang95` are both used as inputs, 
+---------------------------------------------------------------------  
+- `ishape = 4` -- The 95% flux surface values `kappa95` and `triang95` are both used as inputs, 
   and the LCFS values are calculated from them by inverting the equations given above 
-  for ``ishape = 0``.
-
-- `ishape = 5` -- the 95% flux surface values `kappa95` and `triang95` are both used as inputs and 
+  for `ishape = 0`.
+---------------------------------------------------------------------
+- `ishape = 5` -- The 95% flux surface values `kappa95` and `triang95` are both used as inputs and 
   the LCFS values are calculated from a fit to MAST data:
-  
+
   $$
-  \kappa = 0.91 \, \kappa_{95} + 0.39
+  \kappa = 0.913 \, \kappa_{95} + 0.38654
   $$
 
   $$
-  \delta = 0.77 \, \delta_{95} + 0.19 
+  \delta = 0.77394 \, \delta_{95} + 0.18515 
   $$
-
-- `ishape = 6` -- the input values for `kappa` and `triang` are used directly and the 95% flux 
+---------------------------------------------------------------------
+- `ishape = 6` -- The input values for `kappa` and `triang` are used directly and the 95% flux 
   surface values are calculated using the MAST scaling from `ishape = 5`.
-
-- `ishape = 7` -- the 95% flux surface values `kappa95` and `triang95` are both used as inputs and 
+---------------------------------------------------------------------
+- `ishape = 7` -- The 95% flux surface values `kappa95` and `triang95` are both used as inputs and 
   the LCFS values are calculated from a fit to FIESTA runs:
-  
+
   $$
-  \kappa = 0.91 \, \kappa_{95} + 0.39
+  \kappa = 0.90698 \, \kappa_{95} + 0.39467
   $$
 
   $$
-  \delta = 1.38 \, \delta_{95} + 0.05 
+  \delta = 1.3799 \, \delta_{95} + 0.048306 
   $$
-
-- `ishape = 8` -- the input values for `kappa` and `triang` are used directly and the 95% flux 
+---------------------------------------------------------------------
+- `ishape = 8` -- The input values for `kappa` and `triang` are used directly and the 95% flux 
   surface values are calculated using the FIESTA fit from `ishape = 7`.
+---------------------------------------------------------------------
+- `ishape = 9` -- The input values for `triang` and `rli` are used, `kappa` and the 95% flux 
+  surface values are calculated.
+
+  $$
+   \kappa = \left(\left(1.09+\frac{0.26}{l_i}\right)\left(\frac{1.5}{A}\right)^{0.4}\right)
```

Comment:
Its not clear if this also comes from reference [1]?

Reply:
Only the 95% values are. Which is referenced with [1]